vue-frontend/src/views/dashboard/index.vue

```diff
@@ -219,6 +234,27 @@ export default {
                 }
                 _this.ruleForm.usdcBalance = '0.' + String(odd + tokens).replace(/(0+)\b/gi,"")
             }
+        },
+        requestTypeChange(value) {
+            let _this = this
+            let paramsObject = {
+                account: _this.ruleForm.address, 
+                tokens: [], 
+                amounts: []
+            }
+            value.map(item => {
+                switch (item) {
+                    case 'MATIC':
+                        paramsObject.tokens.push(process.env.FAUCET_ADDRESS)
```

Comment:
MATIC address is wrong, it's 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE

FAUCET_ADDRESS is contract address, it's not token